lms/envs/common.py

```diff
@@ -2758,6 +2759,7 @@
 MOBILE_APP_USER_AGENT_REGEXES = [
     r'edX/org.edx.mobile',
 ]
+MOBILE_APP_VERSION_REGEX = r'[0-9]+.[0-9]+.[0-9]+(.[0-9a-zA-Z]*)?'  # X.X.X.Y where X is a number and Y is alphanumeric
```

Comment:
This also needs to be tweaked. It will accept versions with leading zeros e.g. "1.02.003".